config/config.go

```diff
@@ -41,28 +43,30 @@ func Get() (*Config, error) {
 	}
 
 	cfg := &Config{
-		BindAddr:                   ":20000",
-		BabbageURL:                 "http://localhost:8080",
-		RendererURL:                "http://localhost:20010",
-		CookiesControllerURL:       "http://localhost:24100",
-		HomepageControllerURL:      "http://localhost:24400",
-		DatasetControllerURL:       "http://localhost:20200",
-		FilterDatasetControllerURL: "http://localhost:20001",
-		GeographyControllerURL:     "http://localhost:23700",
-		GeographyEnabled:           false,
-		FeedbackControllerURL:      "http://localhost:25200",
-		FeedbackEnabled:            false,
-		SearchControllerURL:        "http://localhost:25000",
-		SearchRoutesEnabled:        false,
-		APIRouterURL:               "http://localhost:23200/v1",
-		DownloaderURL:              "http://localhost:23400",
-		PatternLibraryAssetsPath:   "https://cdn.ons.gov.uk/sixteens/f816ac8",
-		SiteDomain:                 "ons.gov.uk",
-		RedirectSecret:             "secret",
-		SQSAnalyticsURL:            "",
-		ContentTypeByteLimit:       5000000,
-		HealthckeckCriticalTimeout: 90 * time.Second,
-		HealthckeckInterval:        30 * time.Second,
+		BindAddr:                            ":20000",
+		BabbageURL:                          "http://localhost:8080",
+		RendererURL:                         "http://localhost:20010",
+		CookiesControllerURL:                "http://localhost:24100",
+		HomepageControllerURL:               "http://localhost:24400",
+		DatasetControllerURL:                "http://localhost:20200",
+		FilterDatasetControllerURL:          "http://localhost:20001",
+		GeographyControllerURL:              "http://localhost:23700",
+		GeographyEnabled:                    false,
+		FeedbackControllerURL:               "http://localhost:25200",
+		FeedbackEnabled:                     false,
+		SearchControllerURL:                 "http://localhost:25000",
+		SearchRoutesEnabled:                 false,
+		APIRouterURL:                        "http://localhost:23200/v1",
+		DownloaderURL:                       "http://localhost:23400",
+		PatternLibraryAssetsPath:            "https://cdn.ons.gov.uk/sixteens/f816ac8",
+		SiteDomain:                          "ons.gov.uk",
+		RedirectSecret:                      "secret",
+		SQSAnalyticsURL:                     "",
+		ContentTypeByteLimit:                5000000,
+		HealthckeckCriticalTimeout:          90 * time.Second,
+		HealthckeckInterval:                 30 * time.Second,
+		ZebedeeRequestMaximumTimeoutSeconds: 5,
```

Comment:
This should be of type `duration` not int. It should be consistent with the other time based vars in this struct and the name also implies it is a time based value.